tests/gtest/aviftest_helpers.cc

```diff
@@ -186,47 +155,33 @@ bool AreImagesEqual(const avifImage& image1, const avifImage& image2,
   }
   assert(image1.width * image1.height > 0);
 
-  avifPixelFormatInfo info;
-  avifGetPixelFormatInfo(image1.yuvFormat, &info);
-
-  for (int c = 0; c < 4; c++) {
+  for (avifChannelIndex c :
+       {AVIF_CHAN_Y, AVIF_CHAN_U, AVIF_CHAN_V, AVIF_CHAN_A}) {
     if (ignore_alpha && c == AVIF_CHAN_A) continue;
-    uint8_t* row1 =
-        (c == AVIF_CHAN_A) ? image1.alphaPlane : image1.yuvPlanes[c];
-    uint8_t* row2 =
-        (c == AVIF_CHAN_A) ? image2.alphaPlane : image2.yuvPlanes[c];
+    const uint8_t* row1 = avifImagePlane(&image1, c);
+    const uint8_t* row2 = avifImagePlane(&image2, c);
     if (!row1 != !row2) {
       return false;
     }
     if (!row1) {
       continue;
     }
-    const uint32_t row_bytes1 =
-        (c == AVIF_CHAN_A) ? image1.alphaRowBytes : image1.yuvRowBytes[c];
-    const uint32_t row_bytes2 =
-        (c == AVIF_CHAN_A) ? image2.alphaRowBytes : image2.yuvRowBytes[c];
-    const uint32_t plane_width =
-        (c == AVIF_CHAN_Y || c == AVIF_CHAN_A)
-            ? image1.width
-            : ((image1.width + info.chromaShiftX) >> info.chromaShiftX);
-    const uint32_t plane_height =
-        (c == AVIF_CHAN_Y || c == AVIF_CHAN_A)
-            ? image1.height
-            : ((image1.height + info.chromaShiftY) >> info.chromaShiftY);
+    const uint32_t plane_width = avifImagePlaneWidth(&image1, c);
```

Comment:
Optional: Lines 166-168 could be omitted:

```
    if (!row1) {
      continue;
    }
```

You omitted the corresponding code in the two functions that were modified above. It would be good to be consistent. Either form is fine by me.